test/runtests.jl

```diff
@@ -9,6 +9,9 @@ using StaticArrays
 using Test, Random, Plots
 using ReferenceTests, ImageIO
 
+# fix import conflict with Plots
+const BezierCurve = Meshes.BezierCurve
+
```

Comment:
Should we add this constant definition inside the testset? That way we don't need to keep track of the main runtests.jl file when things change in the future.